### PR TITLE
 ENG-8443: add bash commands in bin for snapshotconvert and snapshotverify

### PR DESCRIPTION
**Pull request by JadenW**

### CODE REVIEW

**Comment by vtkstef** [comment]
I would like to ask andrew if this is something that we should bring under the volt or voltadmin family of subcommand

**Comment by vtkstef** [comment]
checked with @ajgent and he is fine with two separate bash scripts

File: bin/snapshotconvert [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# add libs to CLASSPATH
+for f in $VOLTDB_LIB/*.jar; do
+  CLASSPATH=${CLASSPATH}:$f;
+done
+
+CLASSPATH=${CLASSPATH}
+
+# run it
+export CMDVAL="$JAVA -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp -Djava.awt.headless=true $JAVA_HEAP_MAX $VOLTDB_OPTS -Dlog4j.configuration=file://${LOG4J_CONFIG_PATH} "
+export CMDVAL=$CMDVAL"-classpath $CLASSPATH org.voltdb.utils.SnapshotConverter"
```
Comment (vtkstef) [inline bin/snapshotconvert]: have you tried these commands? Shouldn't it be

```bash
export CMDVAL="$CMDVAL-classpath $CLASSPATH org.voltdb.utils.SnapshotConverter"

File: bin/snapshotverifier [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+# add libs to CLASSPATH
+for f in $VOLTDB_LIB/*.jar; do
+  CLASSPATH=${CLASSPATH}:$f;
+done
+
+CLASSPATH=${CLASSPATH}
+
+# run it
+export CMDVAL="$JAVA -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp -Djava.awt.headless=true $JAVA_HEAP_MAX $VOLTDB_OPTS -Dlog4j.configuration=file://${LOG4J_CONFIG_PATH} "
+export CMDVAL=$CMDVAL"-classpath $CLASSPATH org.voltdb.utils.SnapshotVerifier"
```
Comment (vtkstef) [inline bin/snapshotverifier]:
```bash
export CMDVAL="${CMDVAL}-classpath $CLASSPATH org.voltdb.utils.SnapshotVerifier"

File: src/frontend/org/voltdb/utils/SnapshotConverter.java [hunk 3]
```diff
@@ -303,8 +303,7 @@ public static void main(String[] args) {
     }
 
     private static void printHelpAndQuit( int code) {
-        System.out.println("java -cp <classpath> -Djava.library.path=<library path> org.voltdb.utils.SnapshotConverter --help");
-        System.out.println("java -cp <classpath> -Djava.library.path=<library path> org.voltdb.utils.SnapshotConverter --dir dir1 --dir dir2 --dir dir3" +
+        System.out.println("Usage\nsnapshotconverter --dir dir1 --dir dir2 --dir dir3 " +
```
Comment (vtkstef) [inline src/frontend/org/voltdb/utils/SnapshotConverter.java]: why did you omit the line with the `--help` command line option

File: src/frontend/org/voltdb/utils/SnapshotVerifier.java [hunk 4]
```diff
@@ -106,8 +107,8 @@ public static void verifySnapshots(
     }
 
     private static void printHelpAndQuit( int code) {
-        System.out.println("Usage\nSpecific snapshot: java -cp <classpath> -Djava.library.path=<library path> org.voltdb.utils.SnapshotVerifier snapshot_name --dir dir1 --dir dir2 --dir dir3");
-        System.out.println("All snapshots: java -cp <classpath> -Djava.library.path=<library path> org.voltdb.utils.SnapshotVerifier --dir dir1 --dir dir2 --dir dir3");
+        System.out.println("Usage\nSpecific snapshot: snapshotverifier snapshot_name --dir dir1 --dir dir2 --dir dir3");
```
Comment (vtkstef) [inline src/frontend/org/voltdb/utils/SnapshotVerifier.java]: why did you omit the line with the `--help` command line option

**Comment by vtkstef** [comment]
test these commands manually: when the voltdb bin directory is in PATH variable and PWD is somewhere not in the VOLTDB_HOME directory hierarchy. Email your test output for both commands

File: bin/snapshotconvert [hunk 5]
```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
```
Comment (vtkstef) [inline bin/snapshotconvert]: did you set the execute bit permission on the file

```bash
chmod a+x bin/snapshotconvert bin/snapshotverifier
```

File: bin/snapshotverifier [hunk 6]
```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
```
Comment (vtkstef) [inline bin/snapshotverifier]: did you set the execute bit permission on the file?

```bash
chmod a+x bin/snapshotconvert bin/snapshotverifier
```

File: src/frontend/org/voltdb/utils/SnapshotVerifier.java (hunk 4 above)
Comment (JadenW) [inline src/frontend/org/voltdb/utils/SnapshotVerifier.java]: I will add them. Previously SnapshotConverter print it while SnapshotVerifier don't. 
The usage will always print if execution is not successful, I thought I could omit them.

File: bin/snapshotverifier (hunk 6 above)
Comment (JadenW) [inline bin/snapshotverifier]: yes, they are executable.

voltdb@voltdb-SH61R:~/workspace/test$ ls -ltr ~/workspace/voltdb/bin/snapshot*
-rwxrwxr-x 1 voltdb voltdb 4704 Jul  9 16:04 /home/voltdb/workspace/voltdb/bin/snapshotverifier
-rwxrwxr-x 1 voltdb voltdb 4706 Jul  9 16:04 /home/voltdb/workspace/voltdb/bin/snapshotconvert

File: bin/snapshotverifier (hunk 2 above)
Comment (JadenW) [inline bin/snapshotverifier]: I copy this directly from CSVLoader file. 
Test shows this two expressions has same output, but I think your suggestion is good.  It's better bash syntax I suppose? 
Should I address this changes to other files in bin?

**Pull request merged**